composer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "jstoone/html-form-validator",
+    "description": "Validate a form based on it's html data attributes.",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Jakob Steinn",
+            "email": "jakobsteinn@protonmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=5.5.9",
```

Comment:
Never use un-bounded version constraints. Use `^5.5.9|^7.0` instead. I also suggest simply requiring `7.0`, since this is a new package, and you may benefit a lot from `yield from` in this particular library